plugins/microsoft_defender_incidents/help.md

```diff
@@ -0,0 +1,476 @@
+|Recipient|string|False|Available if entityType is MailMessage, containing recipient address|
+|Registry Hive|string|False|Available if entityType is Registry, containing registry hive|
+|Registry Key|string|False|Available if entityType is Registry, containing registry key|
+|Registry Value|string|False|Available if entityType is Registry, containing registry key value|
+|Registry Value Type|string|False|Available if entityType is Registry, containing type of registry value|
+|Security Group ID|string|False|Available if entityType is SecurityGroup, containing security group identifier|
+|Security Group Name|string|False|Available if entityType is SecurityGroup, containing security group name|
+|Sender|string|False|Available if entityType is User/MailBox/MailMessage, containing sender address|
+|SHA1|string|False|Available if entityType is File, containing file hash for alerts associated with a file or process|
+|SHA256|string|False|Available if entityType is File, containing file hash for alerts associated with a file or process|
+|Subject|string|False|Available if entityType is MailMessage, containing mail subject|
+|URL|string|False|Available if entityType is URL, containing URL for alerts associated to network events|
+|User Principal Name|string|False|Available if entityType is User/MailBox/MailMessage, containing user principal name|
+|User SID|string|False|Available if entityType is User, containing user SID|
+
+#### incident
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Alerts|[]alert|False|Array containing all of the alerts related to the incident|
+|Assigned To|string|False|Owner of the incident|
+|Classification|string|False|Specification for the incident, where possible values are - Unknown, FalsePositive, and TruePositive|
+|Comments|[]comment|False|Array of comments created by secops when managing the incident|
+|Created Time|date|False|Time when incident was first created|
+|Detection Source|string|False|Specifies source of detection|
+|Determination|string|False|Specifies the determination of the incident, where possible values are - NotAvailable, Apt, Malware, SecurityPersonnel, SecurityTesting, UnwantedSoftware, and Other|
+|Incident ID|integer|False|Identifier of incident|
+|Incident Name|string|False|String value containing incident's name|
+|Last Updated Time|date|False|Time when incident was last updated on the backend|
+|Severity|string|False|Indicates the possible impact on assets, where possible values are - Informational, Low, Medium, and High|
+|Status|string|False|Categorize incidents, where possible values are - Active, InProgress, Resolved, and Redirected|
+|Tags|[]string|False|Array of custom tags associated with an incident|
+
+
+## Troubleshooting
+
+_This plugin does not contain any troubleshooting information._
+
+# Version History
+
+* 1.0.0 - Initial plugin (Actions: List All Incidents, Get Incident, Update Incident, Triggers: Get New Incidents)
```

Comment:
```suggestion
* 1.0.0 - Initial plugin
```

In general, we don't need to list the actions affected by a version change in the history log. Update logs should be brief and high-level. For a brand new plugin, everything is new so we don't need to list all affected actions.

If there was a change applied to most or all of a plugin's actions, we don't need to list the actions.

If only one or a few actions were affected, for example, in one specific runtime bugfix, then it's helpful to list the action.